project/plugins.sbt

```diff
@@ -14,30 +14,16 @@
  * limitations under the License.
  */
 
+addSbtPlugin("com.github.sbt" % "sbt-release" % "1.1.0")
 
-resolvers += Resolver.url("artifactory", url("http://scalasbt.artifactoryonline.com/scalasbt/sbt-plugin-releases"))(Resolver.ivyStylePatterns)
+addSbtPlugin("com.github.sbt" % "sbt-pgp" % "2.1.2")
 
-resolvers += "bintray-spark-packages" at "https://dl.bintray.com/spark-packages/maven/"
-
-resolvers += "Typesafe Repository" at "http://repo.typesafe.com/typesafe/releases/"
-
-addSbtPlugin("com.github.sbt" % "sbt-release" % "1.0.15")
-
-addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "2.3")
-
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
-
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.9")
-
-addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.3")
```

Comment:
I think we still need other plugins except `sbt-coursier` which should is already in SBT 1.5. Those plugins are for release process.

Reply:
i have added back `addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "2.3")`
that was the only plugin missing (i.e. we have release, sonatype, pgp, assembly now, as required)